aicsimageio/tests/readers/test_ome_tiff_reader.py

```diff
@@ -330,6 +316,86 @@ def test_multi_scene_ome_tiff_reader(
     )
 
 
+@pytest.mark.skip(reason="Planned for 4.1 Levels / Multi-Resolution Support")
```

Comment:
Note, these are skipped. Just adding the files to repo now and showing that we can read the metadata. Hopefully the two issues I create can be patched before we get started on 4.1 work.

Reply:
A lot of comments on my early testing can be found at https://github.com/AllenCellModeling/aicsimageio/issues/140

Reply:
These are no longer skipped as Gohlke told me about the `level` parameter allowing us to read pyramid files and simply default to only reading `level=0`.